lib/fog/utilities.rb

```diff
@@ -1,7 +1,9 @@
 class Hash
   def symbolize_keys!
     keys.each do |key|
-      self[(key.to_sym rescue key)] = delete(key) if key.respond_to?(:to_sym) && !key.is_a?(Fixnum)
+      if key.respond_to?(:to_sym) && !key.is_a?(Integer)
```

Comment:
Sure. The idea was to start using just strings internally, and deprecate symbols as keys.
Then in a future version we would drop it for good. =)

Anyways, this got fixed in #65.